source/_integrations/group.markdown

```diff
@@ -51,6 +52,10 @@ icon:
   description: The icon that shows in the front end.
   required: false
   type: string
+device_class:
+  description: The device class which the group matches.
```

Comment:
Maybe make it clear to the user which device classes you can choose? Link to a page with the list of options?

Reply:
Sorry, I don't know the syntax to link to the binary sensor device list page. Can you help point me in the right direction?

Reply:
```suggestion
  description: The device class which the group matches, list of possible [device classes](/integrations/binary_sensor/#device-class).
```